components/browser/awesomebar/src/main/java/mozilla/components/browser/awesomebar/layout/DefaultSuggestionViewHolder.kt

```diff
@@ -120,29 +116,47 @@ internal sealed class DefaultSuggestionViewHolder {
 internal class IconLoader(
     private val iconView: ImageView
 ) {
-    private val scope = CoroutineScope(Dispatchers.Main)
     @VisibleForTesting
     internal var iconJob: Job? = null
```

Comment:
I will delete this `iconJob` and its corresponding tests once I get the go ahead to move forward with this solution :)